### PR TITLE
New website props layout

### PR DESCRIPTION
**Pull request by Gregjarvez**
#1006 
### version-0.19.0

- [x] avatar.md
- [x] badge.md 
- [x] button.md
- [x] button_group.md
- [x] card.md
- [x] checkbox.md
- [x] divider.md
- [x] forms.md
- [x] header.md
- [x] icon.md
- [x] labels.md
- [x] lists.md
- [x] pricing.md
- [x] rating.md
- [x] searchbar.md
- [x] slider.md
- [x] social_icons.md
- [x] testing.md
- [x] text.md
- [x] tile.md

### version-1.0.0-beta2

- [x] button.md
- [x] button_group.md
- [x] overlay.md
- [x] searchbar.md
- [x] input.md

### version-1.0.0-beta3

- [x] button.md
- [x] card.md
- [x] header.md
- [x] searchbar.md

### CODE REVIEW

**Comment by iRoachie** [comment]
Awesome, love what you did here with the checkboxes. Let's leave the files in the `docs/` for last as they maybe edited by other PRs scheduled for the next release

**Comment by Gregjarvez** [comment]
@iRochie sure i will leave those

**Comment by Gregjarvez** [comment]
@iRoachie I wrote a node script that does this fairly quickly 😉I will get this done asap

**Comment by Gregjarvez** [comment]
@iRoachie looks like we are done with the website folder.

File: website/versioned_docs/version-0.19.0/badge.md [hunk 1]
```diff
@@ -33,3 +33,71 @@ Example badge usage
...
+
+### value
+  text value to be displayed by badge, defaults to empty
+  
+  | Type    | Default |
+  |:-------:|:-------:|
+  | string or number | none |
+  
+  
+### wrapperstyle
```
Comment (iRoachie) [inline website/versioned_docs/version-0.19.0/badge.md]: wrapperstyle to `wrapperStyle`

File: website/versioned_docs/version-0.19.0/divider.md [hunk 2]
```diff
@@ -10,7 +10,12 @@ Example divider usage
 ```
 
 #### Divider props
+  * [style](#style)
```
Comment (iRoachie) [inline website/versioned_docs/version-0.19.0/divider.md]: Missed the `Reference` header all the other pages have

File: website/versioned_docs/version-1.0.0-beta2/input.md [hunk 3]
```diff
@@ -42,15 +42,94 @@ import { Input } from 'react-native-elements';
...
+  * [displayError](#displayerror)
+  * [errorMessage](#errormessage)
+  * [errorStyle](#errorstyle)
+  * [inputStyle](#inputstyle)
+  * [leftIcon](#lefticon)
+  * [leftIconContainerStyle](#lefticoncontainerstyle)
+  * [rightIcon](#righticon)
+  * [rightIconContainerStyle](#righticoncontainerstyle)
+  * [shake](#shake)
+
```
Comment (iRoachie) [inline website/versioned_docs/version-1.0.0-beta2/input.md]: Missed the `Reference` header here too

Comment (Gregjarvez) [reply website/versioned_docs/version-1.0.0-beta2/input.md]: I will double check them later this week @iRoachie

**Comment by iRoachie** [comment]
Thanks again @Gregjarvez, superb PR

**Comment by Gregjarvez** [comment]
@iRoachie here is the simple script I used. Involves alittle bit of manual work but it worked for me :)
https://github.com/Gregjarvez/table-to-list . Hope it is useful 

thanks !!!!!!!

**Pull request merged**